src/lib/routes/proxy-api/index.ts

```diff
@@ -145,7 +145,13 @@ export default class ProxyController extends Controller {
         res: Response<ProxyFeaturesSchema>,
     ) {
         let toggles;
-        if (this.flagResolver.isEnabled('proxyReturnAllToggles')) {
+        const releaseFlag = this.flagResolver.isEnabled(
+            'proxyReturnAllTogglesKillSwitch',
```

Comment:
On the semantics of kill switches: Should they kill something when they're enabled or when they're disabled? 

I thought that kill switches generally meant that a feature is turned off when the kill switch is engaged? At least that's what the [kill switches best practice blog post](https://www.getunleash.io/blog/kill-switches-best-practice) from our blog says. 

I don't know if we've used kill switches anywhere else in our code base. We should probably be consistent internally, but ideally also with the material we put out on how to use it.

So if that's correct, then the `releaseFlag` variable is the opposite of `isEnabled('proxyReturnAllTogglesKillSwitch')`, I think 🤔